kvirt/k3s/bootstrap.sh

```diff
@@ -1,23 +1,22 @@
 {% set extra_args = [] %}
-{% for component in disabled_components %}
-{% do extra_args.append("--disable " + component) %}
+{% for component in k3s_extra_args_all %}
+{% do extra_args.append(component) %}
+{% endfor %}
+{% for component in k3s_extra_args_masters %}
+{% do extra_args.append(component) %}
 {% endfor %}
 apt-get -y install curl
 {% if masters > 1 %}
-curl -sfL https://get.k3s.io | {{ "INSTALL_K3S_EXEC='--flannel-backend=none'" if sdn != "flannel" else '' }} INSTALL_K3S_CHANNEL={{ install_k3s_channel }} INSTALL_K3S_VERSION={{ install_k3s_version if install_k3s_version != "latest" else '' }} K3S_TOKEN={{ token }} sh -s - server --cluster-init {{ extra_args|join(" ") }}
+curl -sfL https://get.k3s.io | INSTALL_K3S_CHANNEL={{ install_k3s_channel }} INSTALL_K3S_VERSION={{ install_k3s_version if install_k3s_version != "latest" else '' }} K3S_TOKEN={{ token }} sh -s - server --cluster-init {{ extra_args|join(" ") }}
 export IP={{ api_ip }}
 {% else %}
-curl -sfL https://get.k3s.io | {{ "INSTALL_K3S_EXEC='--flannel-backend=none'" if sdn != "flannel" else '' }} INSTALL_K3S_CHANNEL={{ install_k3s_channel }} INSTALL_K3S_VERSION={{ install_k3s_version if install_k3s_version != "latest" else '' }} K3S_TOKEN={{ token }} sh -s - server {{ extra_args|join(" ") }}
+curl -sfL https://get.k3s.io | INSTALL_K3S_CHANNEL={{ install_k3s_channel }} INSTALL_K3S_VERSION={{ install_k3s_version if install_k3s_version != "latest" else '' }} K3S_TOKEN={{ token }} sh -s - server {{ extra_args|join(" ") }}
 export IP=$(hostname -I | cut -f1 -d" ")
 {% endif %}
+apt-get -y remove curl
```

Comment:
i find it misleading that one has to install curl as per k3s doc only to perform the install.
What if you want to troubleshoot and you see curl is not there? that would be quite misleading

Reply:
I agree. cURL is not a requirement for K3s itself - but for the installation of it. In the way we do it - the script method. But, still - cURL is not something I want on my K3s nodes, in regards to security.
One could just install cURL again.
I guess it's a matter of how opinionated we allow ourselves to be.